providers/summary/scancode.js

```diff
@@ -33,8 +34,11 @@ class ScanCodeSummarizer {
     const result = {}
     this.addDescribedInfo(result, harvested)
     let declaredLicense = this._readDeclaredLicense(scancodeVersion, harvested)
-    if (!declaredLicense || declaredLicense === 'NOASSERTION') {
-      declaredLicense = this._getDeclaredLicense(scancodeVersion, harvested, coordinates)
+    if (!isDeclaredLicense(declaredLicense)) {
+      declaredLicense = this._readLicenseExpression(harvested) || declaredLicense
```

Comment:
In the latest ScanCode format, "the codebase level packages has been also updated: license_expression -> declared_license_expression, also with it’s SPDX version, declared_license -> extracted_license_statement" from [doc](https://scancode-toolkit.readthedocs.io/en/stable/reference/license-detection-reference.html#chnagelog-summary).  Here in CD with 30.1.0 ScanCode data, declared_license is used to parse package level license information.  In addition, package level license_expression (->declared_license_expression in the new format) can be utilized to provide license information.  Let me know if you have any feedback.